### DISCUSSION TITLE
Table Header Alignment

### DISCUSSION THREAD

**Discussion by ramoneguru** [twilio-labs/paste#1360] [Q&A] [answered]
I have a table element and the headers in it are mostly fine, but I have one header that requires a sub-heading (image below)

<img width="1115" alt="Screen Shot 2021-04-12 at 12 29 01 PM" src="https://user-images.githubusercontent.com/476482/114450721-38c31300-9b8b-11eb-978b-15f3427e2c93.png">

Notice that the headers don't quite align (sub-heading is throwing things off a touch). How can I go about aligning all the headers and still keep that sub-heading text?

**Comment by SiTaggart** [comment, marked as answer]
If we don't already, we might need to expose `vAlign` on the `Th` so you can set vertical align top. 

I think that _should_ do what you're after?

Can you inspect with web tools, manually apply `vertical-align: top` to the `<th />` and we if it gives you the desired effect?

**Reply by ramoneguru** [reply]
Yeah, manually applied to all and then centered the "Capabilites" text (text-align is already exposed). Seems to work

<img width="1113" alt="Screen Shot 2021-04-13 at 12 18 42 PM" src="https://user-images.githubusercontent.com/476482/114608356-763ea380-9c52-11eb-885a-b47862e2142d.png">

**Reply by SiTaggart** [reply]
Actually, we don't need to do anything. You can set `verticalAlign` on the row and the `Th` inherits

https://codesandbox.io/s/double-headers-n2luo?file=/src/index.tsx
